crates/aquascope/src/analysis/stepper/find_steps.rs

```diff
@@ -1017,7 +1063,7 @@ impl<'a, 'tcx: 'a> HirVisitor<'tcx> for HirStepPoints<'a, 'tcx> {
       // TODO: have a split strategy for bare loops. They could be infinite, and
       // thus have no exit block. This shouldn't be an issue but it currently is.
       EK::Loop(_block, _label, LoopSource::Loop, _span) => {
-        todo!("bare loops aren't working yet, sorry!")
+        self.report_unsupported(expr.hir_id, "Bare loops aren't working yet, sorry! Can I interest you in a `for` or `while` loop?");
```

Comment:
Is there a reason that unsupported is distinct from fatal? I assume we don't want to present partial output for a partially supported program.

Reply:
This may just be poor naming. Both are fatal errors and are presented to the users as such. The main differences are logging (internal) and priority. Unsupported items are returned with higher priority because they can also cause fatal items later down the pipeline but we only report one error per body.